modules/claims_api/lib/claims_api/error/soap_error_handler.rb

```diff
@@ -7,7 +7,7 @@ class SoapErrorHandler
     # list of fault codes: https://hub.verj.io/ebase/doc/SOAP_Faults.htm
     # list faraday errors: https://lostisland.github.io/faraday/middleware/raise-error
 
-    def handle_errors(response)
+    def handle_errors(response, _endpoint = nil)
```

Comment:
why did we add `_endpoint` here if it's unused?

Reply:
thought i removed it. will do that now